qa/tasks/cephfs/test_client_recovery.py

```diff
@@ -601,3 +601,21 @@ def test_dont_mark_unresponsive_client_stale(self):
         self.assertLess(time.time(), time_at_beg + SESSION_AUTOCLOSE)
 
         self.mount_a.send_signal('sigcont')
+
+    def test_config_session_timeout(self):
+        self.fs.mds_asok(['config', 'set', 'mds_defer_session_stale', 'false'])
+        session_timeout = self.fs.get_var("session_timeout")
+        mount_a_gid = self.mount_a.get_global_id()
+
+        self.fs.mds_asok(['session', 'config', '%s' % mount_a_gid, 'timeout', '%s' % (session_timeout * 2)])
+
+        self.mount_a.kill();
+        self.mount_a.kill_cleanup()
```

Comment:
`kill_cleanup` now waits for the hard reboot to complete: #28825 

I guess the right thing to do here is to add a sleep after `kill()`, assert 2 sessions, then do `kill_cleanup()`.

See this failure:

/ceph/teuthology-archive/pdonnell-2019-07-26_06:38:30-kcephfs-wip-pdonnell-testing-20190726.021409-distro-basic-smithi/4152101/teuthology.log